src/module.ts

```diff
@@ -19,7 +51,7 @@ export default defineNuxtModule<StrapiOptions>({
     version: 'v4',
     cookie: {}
   },
-  setup (options: StrapiOptions, nuxt: Nuxt) {
+  setup (options: ModuleOptions, nuxt: Nuxt) {
```

Comment:
Tip: You don't need types here. They are already inferred by `defineNuxtModule`

Reply:
Didn't know that, should we remove it then?